presto-geospatial-toolkit/src/main/java/com/facebook/presto/geospatial/GeometryUtils.java

```diff
@@ -152,45 +223,57 @@ public static OGCGeometry deserialize(Slice shape)
         }
         BasicSliceInput input = shape.getInput();
 
-        int spatialReferenceId = input.readInt();
-        SpatialReference spatialReference = null;
-        if (spatialReferenceId != SPATIAL_REFERENCE_UNKNOWN) {
-            spatialReference = SpatialReference.create(spatialReferenceId);
-        }
-
         // GeometryCollection: spatialReferenceId|len-of-shape1|bytes-of-shape1|len-of-shape2|bytes-of-shape2...
```

Comment:
this comments needs updating: delete spatialReferenceId field and add geometry type field